test/integration/lib.sh

```diff
@@ -53,18 +53,39 @@ kpatch_rhel_dependencies()
 {
 	local kernel_version
 	local arch
+	local rhel_major
+	local py_version
 	kernel_version=$(uname -r)
 	arch=$(uname -m)
+	rhel_major=${VERSION_ID%%.*}
+
+	if [ $rhel_major -ge 8 ]; then
```

Comment:
We try to keep these free of [shellcheck](https://www.shellcheck.net/) warnings (although I see I've already failed at that:), so please enclose "$rhel_major" in quotes, also {}s to be consistent with the rest of the file.

Reply:
Good point, the warnings from these commits should be fixed now!

Reply:
Should this file be added to the list of files checked by "make check"?

Reply:
@jpoimboe I've made a pull request to add this and other bash files to make check #1110